framework/src/modules/token/method.ts

```diff
@@ -640,6 +643,10 @@ export class TokenMethod extends BaseMethod {
 		receivingChainID: Buffer,
 		fee: bigint,
 	): Promise<void> {
+		if (fee < BigInt(0)) {
+			throw new Error('invalid Message Fee');
```

Comment:
I hate to be that guy, but the letter casing on this error message looks weird 🙂 

```suggestion
			throw new Error('Invalid message fee');
```

Reply:
haha I also has this struggle when I wrote this because I saw another error message using small letters. Then I just realised that error indeed did not start with small letter 😂

Fixed